src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String data = readFromFile(fromFileName);
+        String report = createReport(data);
+        writeDataToFile(report, toFileName);
+    }
+
+    private String readFromFile(String fromFileName) {
+        StringBuilder inputString = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                inputString.append(line).append("\n");
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file", e);
+        }
+        return inputString.toString();
+    }
+
+    private String createReport(String data) {
+        final String BU = "buy";
+        final String SU = "supply";
+        final int ZE = 0;
+        final int ON = 1;
```

Comment:
constants are `private static final` class-level values
dont use shortenings, better use full informative names

Reply:
what is the point of private constants in a private method? Maven does not allow me to put long names of constants